### PR TITLE
Button for Downloading CSV: Submission status table - Assessment Tab

### PR DESCRIPTION
**Pull request by dmcavoy**
Added button for downloading csv of the submission status table in the assessment tab of the teacher dashboard.

screen shot 2018-12-19 at 3 51 50 pm

### CODE REVIEW

File: apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js [hunk 1]
```diff
@@ -558,6 +558,26 @@ export const getStudentsMCSummaryForCurrentAssessment = (state) => {
   return studentsSummaryArray;
 };
 
+/**
+ * @returns {array} of objects with keys corresponding to columns
+ * of CSV to download. Columns are defined as CSV_SUBMISSION_STATUS_HEADERS in SubmissionStatusAssessmentsContainer
+ */
+export const getExportableSubmissionStatusData = (state) => {
+
```
Comment (maddiedierker) [inline apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js]: nit: delete extra line before method content

File: apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js [hunk 2]
```diff
@@ -734,6 +754,7 @@ export const getExportableAssessmentData = (state) => {
   return responses;
 };
 
+
```
Comment (maddiedierker) [inline apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js]: nit: delete extra line between methods

File: apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsContainer.jsx [hunk 3]
```diff
@@ -2,19 +2,61 @@ import React, {Component, PropTypes} from 'react';
...
     studentOverviewData: PropTypes.arrayOf(studentOverviewDataPropType),
+    studentExportableData: PropTypes.arrayOf(studentExportableDataPropType),
   };
 
   render() {
+    const {studentExportableData} = this.props;
+
     return (
       <div>
+        <div style={styles.buttonContainer}>
```
Comment (maddiedierker) [inline apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsContainer.jsx:45]: nit: indent the content inside this div for readability

**Review by maddiedierker — APPROVED** [review]
great job! the only feedback comments i have are small code-style nitpicks

File: apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsContainer.jsx (hunk 3 above)
Comment (dmcavoy) [reply apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsContainer.jsx:45]: Updated!

File: apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js (hunk 2 above)
Comment (dmcavoy) [reply apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js]: Updated!

File: apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js (hunk 1 above)
Comment (dmcavoy) [reply apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js]: Updated!

File: apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsContainer.jsx [hunk 4]
```diff
@@ -2,19 +2,61 @@ import React, {Component, PropTypes} from 'react';
...
 
 class SubmissionStatusAssessmentsContainer extends Component {
   static propTypes = {
     studentOverviewData: PropTypes.arrayOf(studentOverviewDataPropType),
+    studentExportableData: PropTypes.arrayOf(studentExportableDataPropType),
   };
 
   render() {
+    const {studentExportableData} = this.props;
+
```
Comment (nkiruka) [inline apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsContainer.jsx]: nit: delete extra space between const and return

Comment (dmcavoy) [reply apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsContainer.jsx]: Updated!

**Review by nkiruka — APPROVED** [review]
The button looks great!!! This is awesome!
celebrate

**Pull request merged**